### PR TITLE
Added options propagation

### PR DESCRIPTION
**Pull request by Xflofoxx**
Added options propagation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,85 @@
 {
-  "name": "decompress-tarbz2",
-  "version": "4.1.0",
-  "description": "decompress tar.bz2 plugin",
-  "license": "MIT",
-  "repository": "kevva/decompress-tarbz2",
+  "_args": [
```
Comment (SamVerschueren) [inline package.json]: Revert the `package.json`

File: readme.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ decompress('unicorn.tar.gz', 'dist', {
 
 ## API
 
-### decompressTarbz()(input)
+### decompressTarbz()(buf)
```
Comment (SamVerschueren) [inline readme.md]: Revert the name to `input` and add an extra `options` parameter

`decompressTarbz()(input, [options])`

Also document the new `options` object by pointing to the `options` parameter of `decompress-tar`. The last one has to wait untill the other PR is merged though.

**Review by SamVerschueren — CHANGES_REQUESTED** [review]

File: readme.md [hunk 3]
```diff
@@ -39,6 +39,10 @@ Type: `Buffer` `Stream`
 Buffer to decompress.
 
 
+#### options
+
+See [decompress-tar](https://github.com/kevva/decompress-tar#options) module options.
+
```
Comment (SamVerschueren) [inline readme.md:45]: Add extra line here.

**Review by SamVerschueren — CHANGES_REQUESTED** [review]

File: readme.md (hunk 3 above)
Comment (Xflofoxx) [reply readme.md:45]: Done

**Comment by sindresorhus** [comment]
Closing as `decompress-tar` doesn't have any options.

**Pull request closed without merging**